index.js

```diff
@@ -11,6 +11,8 @@ async function run() {
 		const releaseTemplate = core.getInput('template');
 		const commitTemplate = core.getInput('commit-template');
 		const exclude = core.getInput('exclude');
+		let dateFormat = core.getInput('date-format');
+		dateFormat = ['relative', 'local', 'default', 'iso', 'rfc', 'short', 'raw'].includes(dateFormat) ? ('--date=' + dateFormat) : ('--date=format:' + dateFormat);
```

Comment:
Don’t pass a whole cli flag to the generateReleaseNotes function, you can create the flag inside it. 
```suggestion
		dateFormat = dateFormat.includes('%') ? 'format:' + dateFormat : dateFormat;
```

Reply:
There should be also `--date=` implemented.
So it will be:
```js
dateFormat = dateFormat.includes('%') ? '--date=format:' + dateFormat : '--date=' + dateFormat;
```

Reply:
Read my comment above the suggestion

Reply:
Which one do you mean?

Reply:
> Don’t pass a whole cli flag to the generateReleaseNotes function, you can create the flag inside it.

Reply:
OK, I will edit it...